test/onnx/test_onnxscript_runtime.py

```diff
@@ -0,0 +1,132 @@
+# Owner(s): ["module: onnx"]
+
+"""Test the support on onnxscript in PyTorch-ONNX converter."""
+from typing import List
+
+import onnx_test_common
+import onnxscript
+import torch
+from onnxscript.onnx_types import FLOAT
+from torch.onnx._internal import jit_utils
+from torch.testing._internal import common_utils
+
+
+class TestONNXScriptRuntime(onnx_test_common._TestONNXRuntime):
+
+    # opset version is
+    # 1. local function is supported after opset 15
+    # 2. onnx-script requires users to determine opset in local function
+    opset_version = 15
+
+    def test_selu_from_onnxscript_example(self):
+
+        x = torch.randn(1, 2, 3, 4, requires_grad=True)
+        model = torch.nn.SELU()
+
+        from onnxscript.onnx_opset import opset15 as op
+
+        # custom domain is needed for custom Op domain name should be
+        # aligned to the one in symbolic_fn
+        # TODO(titaiwang): make an official domain for onnxscript usage
+        custom_opset = onnxscript.values.Opset(domain="onnx-script", version=1)
+
+        @onnxscript.script(custom_opset)
+        def Selu(X, alpha: float, gamma: float):
```

Comment:
This looks odd/confusin, though there is a comment below. I would suggest dropping the parameters alpha/gamma for now.

Reply:
Done